cmd/neofs-cli/modules/container/delete.go

```diff
@@ -92,10 +92,12 @@ Only owner of the container has a permission to remove container.`,
 }
 
 func initContainerDeleteCmd() {
-	commonflags.Init(deleteContainerCmd)
-
 	flags := deleteContainerCmd.Flags()
 
+	flags.StringP(commonflags.WalletPath, commonflags.WalletPathShorthand, commonflags.WalletPathDefault, commonflags.WalletPathUsage)
```

Comment:
Have already you tested it? We bind flags https://github.com/nspcc-dev/neofs-node/blob/master/cmd/neofs-cli/internal/commonflags/flags.go#L61 for the whole container command to make sure they can be used from the configuration. I am not sure about the behaviour if the flag is not found.

Reply:
Yes, here is the log:
```
annikifa@annikiforov:~/workspace/neofs-dev-env$ neofs-cli --rpc-endpoint 192.168.130.71:8080 --wallet wallets/wallet.json container list
Enter password > 
4xqkG3gbgCrYWtbAJThv2TrdYxHR9QyQBc4mkcQAPKTZ
annikifa@annikiforov:~/workspace/neofs-dev-env$ 
annikifa@annikiforov:~/workspace/neofs-dev-env$ 
annikifa@annikiforov:~/workspace/neofs-dev-env$ neofs-cli --rpc-endpoint 192.168.130.71:8080 --wallet wallets/wallet.json container delete --cid 4xqkG3gbgCrYWtbAJThv2TrdYxHR9QyQBc4mkcQAPKTZ --await
Enter password > 
container delete method invoked
awaiting...
container has been removed: 4xqkG3gbgCrYWtbAJThv2TrdYxHR9QyQBc4mkcQAPKTZ
annikifa@annikiforov:~/workspace/neofs-dev-env$ neofs-cli container delete --help
Delete existing container. 
Only owner of the container has a permission to remove container.

Usage:
  neofs-cli container delete [flags]

Flags:
      --address string        address of wallet account
      --await                 block execution until container is removed
      --cid string            container ID
      --force                 do not check whether container contains locks and remove immediately
  -h, --help                  help for delete
  -r, --rpc-endpoint string   remote node address (as 'multiaddr' or '<host>:<port>')
      --session string        path to a JSON-encoded container session token
      --ttl uint32            TTL value in request meta header (default 2)
  -w, --wallet string         path to the wallet or binary key
  -x, --xhdr strings          Request X-Headers in form of Key=Value

Global Flags:
  -c, --config string   config file (default is $HOME/.config/neofs-cli/config.yaml)
  -v, --verbose         verbose output
annikifa@annikiforov:~/workspace/neofs-dev-env$
```